source/help/settings/account-settings.md

```diff
@@ -61,8 +61,13 @@ A notification sound plays for all Mattermost posts that would fire a desktop no
 #### Email Notifications
 Email notifications are sent for mentions and direct messages after you’ve been offline for more than 60 seconds or away from Mattermost for more than 5 minutes. Change the email where notifications are sent from **Account Settings > General > Email**.
 
-#### Mobile Push Notifications
-Push notifications are sent to your mobile device if you have the Android or iOS app installed. They are sent even if you are online, similar to desktop notifications. By default, push notifications are sent "For mentions and direct messages". If your System Administrator has not set up push notifications, this setting will be disabled.
+#### Send Mobile Push Notifications
+Push notifications can be sent to your mobile device if you have the Android or iOS app installed. You can choose the type of activity that will send a notification. By default, push notifications are sent "For mentions and direct messages". If push notifications are sent "Never", the setting to trigger push notifications depending on your status is hidden. If your System Administrator has not set up push notifications, this setting will be disabled.
```

Comment:
"the setting to trigger push notifications depending on your status is hidden"

in Mattermost settings or device settings?